tdr-admins/tdr-user-administrator.md

```diff
@@ -19,7 +19,10 @@ TDR user administrators have rights and privileges to manage:
 1. Contact TDR team to request set up as a TDR user administrator: *[email address and other contact details to be confirmed]*
 2. You will receive an email from the TDR team with:
   * your user name
-  * URL to the Keycloak application
+  * URL to the Keycloak application:
+    * *Intg*: https://auth.tdr-integration.nationalarchives.gov.uk/auth/admin/tdr/console
+    * *Staging*: https://auth.tdr-staging.nationalarchives.gov.uk/auth/admin/tdr/console
+    * *Production*: https://auth.tdr.nationalarchives.gov.uk/auth/admin/tdr/console
```

Comment:
Sorry, I've approved this, but now I'm wondering whether it makes sense to include the staging and intg URLs in these docs? Will it be understood by the users who need these docs, like Claire? Or should we just include the prod URL?